op-batcher/batcher/config.go

```diff
@@ -67,7 +67,8 @@ type CLIConfig struct {
 	// Type of compressor to use. Must be one of [compressor.KindKeys].
 	Compressor string
 
-	Stopped bool
+	Stopped        bool
+	CheckRecentTxs bool
```

Comment:
Would be good to follow the convention in this CLI struct to describe briefly what this does (and take the opportunity to also do this for `Stopped`).

I also suggest we add another feature flag like `WaitNodeSync` to enable the node sync waiting with `WaitRollupSync` at all.

```suggestion
    // If Stopped is true, the batcher starts stopped and won't start batching right away.
    // Batching needs to be started via an admin RPC.
	Stopped bool
	
	// Whether to wait for the sequencer to sync to a recent block at startup.
	WaitNodeSync bool
	
	// Whether to check recent batcher transactions when waiting for node sync at startup.
	// If false, the batcher will just use the current head.
	CheckRecentTxs bool
```